templates/api-template-python/swagger/swagger.json

```diff
@@ -13,6 +13,7 @@
     "paths": {
         "/{service_name}": {
             "options": {
+                "operationId": "{operationId}",
```

Comment:
Can we do the same changes to swagger file in `go` template [here](https://github.com/tmobile/jazz/blob/develop/templates/api-template-go/swagger/swagger.json)?

Remove `x-amazon-apigateway-integration` section and add `operationId `.

Reply:
fixed